setup.py

```diff
@@ -13,7 +13,7 @@
 
 setup(
     name='tr55',
-    version='0.1.0.dev2',
+    version='0.1.0.dev3',
```

Comment:
We shouldn't include the version changes here or in the changlog.  Let's add the new functionality and do a separate release and commit when we're ready.  Also, we've dropped the `dev*` suffix.  Since this has breaking changes, we should release it as `0.2.0` when we do.